atlas-fastq-provider-functions.sh

```diff
@@ -1311,7 +1311,15 @@ fetch_library_files_from_ena() {
     fi
     
     mkdir -p $outputDir
-    cp -a $tempdir/. $outputDir/
+    
+    echo "Files to be copied"
+    echo "${filenames_arr[@]}"
+    
+    for basefile in "${filenames_arr[@]}"; do
```

Comment:
I am wondering if we should use `uniq[@]` instead of `filenames_arr[@]`

Reply:
Agreed..